[2024-10-08T15:43:10.013+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-08T15:43:10.035+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: update_database.test_postgres_connection manual__2024-10-08T15:43:08.170470+00:00 [queued]>
[2024-10-08T15:43:10.045+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: update_database.test_postgres_connection manual__2024-10-08T15:43:08.170470+00:00 [queued]>
[2024-10-08T15:43:10.046+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-10-08T15:43:10.058+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): test_postgres_connection> on 2024-10-08 15:43:08.170470+00:00
[2024-10-08T15:43:10.067+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1219) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-08T15:43:10.068+0000] {standard_task_runner.py:63} INFO - Started process 1221 to run task
[2024-10-08T15:43:10.069+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'update_database', 'test_postgres_connection', 'manual__2024-10-08T15:43:08.170470+00:00', '--job-id', '3852', '--raw', '--subdir', 'DAGS_FOLDER/update.py', '--cfg-path', '/tmp/tmp2xf2a553']
[2024-10-08T15:43:10.070+0000] {standard_task_runner.py:91} INFO - Job 3852: Subtask test_postgres_connection
[2024-10-08T15:43:10.112+0000] {task_command.py:426} INFO - Running <TaskInstance: update_database.test_postgres_connection manual__2024-10-08T15:43:08.170470+00:00 [running]> on host 1fee8bc549a8
[2024-10-08T15:43:10.193+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='update_database' AIRFLOW_CTX_TASK_ID='test_postgres_connection' AIRFLOW_CTX_EXECUTION_DATE='2024-10-08T15:43:08.170470+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-08T15:43:08.170470+00:00'
[2024-10-08T15:43:10.195+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-08T15:43:10.202+0000] {logging_mixin.py:188} INFO - <sqlalchemy.engine.cursor.LegacyCursorResult object at 0x7fd6517fcb60>
[2024-10-08T15:43:10.203+0000] {logging_mixin.py:188} INFO - <class 'sqlalchemy.engine.cursor.LegacyCursorResult'>
[2024-10-08T15:43:10.203+0000] {logging_mixin.py:188} INFO - (datetime.date(2024, 5, 19), datetime.time(15, 0), 'Ross County')
[2024-10-08T15:43:10.204+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-08T15:43:10.204+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-08T15:43:10.212+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=update_database, task_id=test_postgres_connection, run_id=manual__2024-10-08T15:43:08.170470+00:00, execution_date=20241008T154308, start_date=20241008T154310, end_date=20241008T154310
[2024-10-08T15:43:10.242+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-08T15:43:10.259+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-08T15:43:10.261+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
